README.md

```diff
@@ -1 +1,261 @@
+  - generateName: test
+    rules:
+    - apiGroups: [""]
+      # at the HTTP level, the name of the resource for accessing Secret
+      # objects is "secrets"
+      resources: ["secrets"]
+      verbs: ["get", "watch", "list"]
+    subjects:
+    - kind: Group
+      name: manager  # Name is case sensitive
+      apiGroup: rbac.authorization.k8s.io
+```
+
+The reconciliation process will verify the below steps:
+1. It will check if any ScopeInstance CRs reference to ScopeTemplate name or not.
+2. If it is referencing then the clusterRole defined in the scopeTemplate will be created if it does not exist. The created clusterRole will include an ownerReference to thes copeTemplate CR.
+3. If no scopeInstance references the scopeTemplate, the clusterRole defined in the scopeTemplate will be deleted if it does not exist.
+
+
+### ScopeInstance CRD
+
+The ScopeInstance CRD used to define list of `namespaces` that the RBAC in  ScopeTemplate will be created in. A cluster admin will create the scopeInstance CR and will specify:
+
+- The name of a scopeTemplate which defines the RBAC required by the operator
+- A set of namespaces that the operator should be scoped to.
+
+```
+apiVersion: operators.io.operator-framework/v1
+kind: ScopeInstance
+metadata:
+  name: scopeinstance-sample
+spec:
+  scopeTemplateName: scopetemplate-sample
+  namespaces:
+  - default
+```
+
+The reconciliation process will verify the below steps:
+
+1. It will look for ScopeTemplate that ScopeInstance is referencing. if it is not referencing then throw an error with the appropriate message.
+2. If it is referencing and if the `namespaces` array is empty, a single `clusterRoleBinding` will be created. Otherwise, a `roleBinding` will be created in each of the `namespaces`. These resources will include an ownerReference to the ScopeInstance CR.
+
+## Run the Operator Locally
+
+### 1. Run locally outside the cluster 
+
+First, install newly created ScopeInstance and ScopeTemplate CRs
+
+```
+make install
+```
+
+It will create CRDs and throw the below message
+
+```
+customresourcedefinition.apiextensions.k8s.io/scopeinstances.operators.io.operator-framework created
+customresourcedefinition.apiextensions.k8s.io/scopetemplates.operators.io.operator-framework created
+```
+
+Then, run the `oria-operator` with below command and apply ScopeInstance and ScopeTemplate CRDs
+
+```
+make run
+```
+
+Apply ScopeTemplate CRD as below. This will create scopetemplate-sample.
+
+```
+$ kubectl apply -f config/samples/operators_v1_scopetemplate.yaml
+scopetemplate.operators.io.operator-framework/scopetemplate-sample created
+```
+
+Now, create ScopeInstance CRD as below. This will create scopeinstance-sample.
+
+```
+$ kubectl apply -f config/samples/operators_v1_scopeinstance.yaml
+scopeinstance.operators.io.operator-framework/scopeinstance-sample created
+```
+
+Once scopeinstance-sample is created, it will trigger the reconciliation process of ScopeTemplate and ScopeInstance controllers.
```

Comment:
Nit:
```suggestion
Once `scopeinstance-sample` is created, it will trigger the reconciliation process of `ScopeTemplate` and `ScopeInstance` controllers.
```